chord_metadata_service/mohpackets/data/data_converter.py

```diff
@@ -52,16 +54,21 @@ def set_foreign_keys(path):
         with open(input_path, "r") as f:
             data_without_relationships = json.load(f)
 
+        if model == "programs":
+            # Update the values in Program.json
+            for item in data_without_relationships:
+                item["program_id"] = item["program_id"].replace("SYNTHETIC-", program_id)
+
```

Comment:
You don't need to add custom logic here. It should be done through the relationship.json template

Reply:
Fixed.